src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/Shell.java

```diff
@@ -368,7 +369,27 @@ private void processCommandEntry()
    }
 
    public void onSendToConsole(final SendToConsoleEvent event)
-   {  
+   {
+      server_.adaptToLanguage(
+            event.getLanguage(),
+            new ServerRequestCallback<Void>()
+            {
+               @Override
+               public void onResponseReceived(Void response)
+               {
+                  sendToConsoleImpl(event);
+               }
+
+               @Override
+               public void onError(ServerError error)
+               {
+                  Debug.logError(error);
```

Comment:
I think we should still send to the console here as a failsafe; otherwise sending things to the console can be broken indefinitely if something gets broken in language adaptation.